packages/tasks/src/hardware.ts

```diff
@@ -168,6 +176,14 @@ export const SKUS = {
 				tflops: 9.22,
 				memory: [6],
 			},
+			"GTX 1080 Ti": {
+				tflops: 11.3,
+				memory: [12],
+			},
+			"GTX 1070 Ti": {
+				tflops: 1.62,
```

Comment:
do you have a source?

Reply:
@pcuenca Must have done this one wrong just like the 1080, im so sorry!

Reply:
I think I looked at a fp32 source by accident.